### PR TITLE
Analytics Spec

### PR DESCRIPTION
**Pull request by SamDelaney**

### CODE REVIEW

File: docs/analytics_spec.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+## Events
+
+### Auction Viewed
+Triggers when the "View Auction" page is opened.
+
+__Event Properties__
+```typescript
+{
+  type: "AuctionViewed",
+  unitId: unit,
```
Comment (euonymos) [inline docs/analytics_spec.md:75]: Is that `unitId` sort of event identifier?

File: docs/analytics_spec.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+{
+  type: "AuctionAnalyticsViewed",
+  unitId: unit,
+  walletAddr: Address | undefined,
+  userType: userType,
+}
+```
+
+### Auctions Announced
+Triggers when the `announceAuction` transaction succeeds.
```
Comment (euonymos) [inline docs/analytics_spec.md:95]: Do we have an auction identifier here somewhere?

**Review by euonymos — CHANGES_REQUESTED** [review]
LGTM @SamDelaney, dropped a couple of question.

I think we have enough events to begin with, probably it's time to start moving from the other end, I mean reports and embedded widgets we are going to implement. From what comes to my mind:
* Delegate group performance (# auctions completed, # of auctions aborted)
* Seller statistics (#auctions completed)
* Bidder statistics (# bids, # wins, #failures to purchase)
* Lot summary (# of auctions, lowest and highest prices?)

**Comment by SamDelaney** [comment]
@euonymos addressed your comments. Further development of the spec will require events & a demo project for testing, so I plan on closing this PR and revisiting later, barring further feedback.

**Pull request merged**